apps/site/lib/site_web/views/schedule/stop_list.ex

```diff
@@ -125,6 +125,12 @@ defmodule SiteWeb.ScheduleView.StopList do
     ]
   end
 
+  # Rapid transit list data returns in a list with each stop
+  # This data is rendered in front-end so render nothing here
+  def display_departure_range(data) when is_list(data) do
```

Comment:
I can try to refactor it.  Though I have doubts it will be as clean for this is the parent call: `hours: HTML.safe_to_string(ScheduleView.render("_hours_of_op.html", conn.assigns))`